cypress/integration/switchinPages/pages.js

```diff
@@ -1,11 +0,0 @@
-export function switchingPages() {
-  const time = 3000;
-  const baseURL = "http://localhost:3000/random";
-  cy.visit(baseURL);
-
-  cy.intercept("GET", "http://localhost:5000/recommendations").as("switching");
-  cy.wait(time);
-  cy.contains("Home").click();
-  cy.wait("@switching");
-  cy.get("#name");
-}
```

Comment:
Não estão sendo feitos testes com os valores, todo it necessariamente precisa ter um `expect()`, pois ele é justamente o teste que será feito. 😉